pkg/skaffold/jib/jib_maven.go

```diff
@@ -32,26 +32,26 @@ const MinimumJibMavenVersion = "1.4.0"
 // MavenCommand stores Maven executable and wrapper name
 var MavenCommand = util.CommandWrapper{Executable: "mvn", Wrapper: "mvnw"}
 
-// GetDependenciesMaven finds the source dependencies for the given jib-maven artifact.
+// getDependenciesMaven finds the source dependencies for the given jib-maven artifact.
 // All paths are absolute.
-func GetDependenciesMaven(ctx context.Context, workspace string, a *latest.JibMavenArtifact) ([]string, error) {
-	deps, err := getDependencies(workspace, getCommandMaven(ctx, workspace, a), a.Module)
+func getDependenciesMaven(ctx context.Context, workspace string, a *latest.JibArtifact) ([]string, error) {
```

Comment:
Doesn't this still need to get exported? It's used in `local.go` and `types.go`. I also don't know how anything works in go.

Reply:
They were changed to use `GetDependencies()`.